sacc/tracers.py

```diff
@@ -666,7 +666,7 @@ class QPNZTracer(BaseTracer, tracer_type='QPNZ'):
         The qp.ensemble in questions
     """
 
-    def __init__(self, name, ens, **kwargs):
```

Comment:
This is a breaking change right now. I there a way to make the z optional? If this is easy and elegant then maybe do it, otherwise not super necessary, I think. Maybe ask the RAIL people if they're using this yet?

Reply:
Hi @joezuntz ! Have a look at the last push. I made z a kwarg. Most likely the z grid will be contained in the ensemble but for some more exotic ensemble representations it might have to be provided externally to avoid an error. Hope this is satisfactory! If so please feel free to merge.